### ISSUE TITLE
[Research] OUI Compliance audit of `src/core/public/styles`

### ISSUE THREAD

**Issue by BSFishy** [opensearch-project/OpenSearch-Dashboards#4117]
Tracking issue for OUI compliance audit of the `src/core/public/styles` folder.

Follow the steps in #3945 to complete the audit, then open another issue with findings. Once the work of performing the audit is done, this issue can be closed.

Note that this is a core service, which may be involved with the implementation and inclusion of OUI styles and themes. Since this is a core service, the conclusion of this audit might actually not involve any additional work.

<details>
  <summary>Related files</summary>

size (bytes) | file | notes
---|---|---
5.0K | [src/core/public/styles/_ace_overrides.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/core/public/styles/_ace_overrides.scss) | 
1.6K | [src/core/public/styles/_base.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/core/public/styles/_base.scss) | 
45 | [src/core/public/styles/_index.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/core/public/styles/_index.scss) | 

</details>

If this issue isn't already assigned to someone and you'd like to take on this work, please ping @BSFishy in the comments and I can assign it to you.

**Comment by curq** [comment]
@BSFishy Can I take this one please?

**Comment by curq** [comment]
@BSFishy Finished https://github.com/opensearch-project/OpenSearch-Dashboards/issues/4167
